frontend/src/stores/ParticipantStore.js

```diff
@@ -108,6 +117,18 @@ export class ParticipantStore {
     this.routeToQueueTable = data
   }
   // Participant State Actions
+  @action handleParticipantChange = event => {
```

Comment:
Can you pass `e.target` to handleParticipantChange where it is called? 

then in handleParticipantChange, you can de structure  `const { name, value } = target`

and assign as such.
`this.participant[name] = value` 

unless there is a reason to do otherwise, there will be less repetition